datadog_checks_dev/datadog_checks/dev/tooling/requirements.py

```diff
@@ -60,24 +60,6 @@ def __eq__(self, other):
         except (AttributeError, TypeError):
             return NotImplemented
 
-    def __cmp__(self, other):
```

Comment:
methods have been there since python 2.1, we shouldn't need this: https://docs.python.org/2/reference/datamodel.html#object.__ge__